src/main/java/com/alkemy/ong/infrastructure/rest/request/UpdateUserRequest.java

```diff
@@ -0,0 +1,27 @@
+package com.alkemy.ong.infrastructure.rest.request;
+
+import com.alkemy.ong.infrastructure.rest.request.validation.CharactersWithWhiteSpaces;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Size;
+import lombok.Getter;
+import org.springframework.lang.Nullable;
+
+@Getter
+public class UpdateUserRequest {
```

Comment:
Todos los NotEmpty dicen "This field...", cuando se devuelve el mensaje de error, ese error no se sabe a que atributo de la request pertenece.